tests/wallet/cat_wallet/test_trades.py

```diff
@@ -422,11 +393,8 @@ async def test_cat_trades(
 
     driver_dict: Dict[bytes32, PuzzleInfo] = {}
     for wallet in (cat_wallet_maker, new_cat_wallet_maker):
-        asset_id: str = wallet.get_asset_id()
```

Comment:
was there a warning about this type annotation?
it has the effect that it protects against `get_asset_id()`'s return value changing